myapp/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "myapp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "dependencies": {
+    "express": "^4.15.3"
```

Comment:
Let's move this `express` dependency into the `package.json` file in the root directory, then remove this `package.json` file.